src/Microsoft.DotNet.ProjectModel/Files/PackIncludeEntry.cs

```diff
@@ -26,18 +32,16 @@ public PackIncludeEntry(string target, string[] sourceGlobs, int line, int colum
             Column = column;
         }
 
-        private static string[] ExtractValues(JsonValue json)
+        private static string[] ExtractValues(JToken json)
```

Comment:
This seems like a converter method. `JTokenConverter.ToStringArray(JToken json)`?